roles/monitor/tasks/main.yml

```diff
@@ -33,4 +33,22 @@
   template:
     src: monitor.nginx.j2
     dest: /etc/nginx/sites-enabled/monitor    
-  notify: restart nginx
\ No newline at end of file
+  notify: restart nginx
+
+- name: configure kibana
+  template:
+    src: monitor.kibana.j2
+    dest: /etc/nginx/sites-enabled/kibana    
+  notify: restart nginx
```

Comment:
we will restart nginx 3 times, may not add a notify handler, just add a restart action after vhosts files copy